hera_cal/lstbin.py

```diff
@@ -714,10 +732,11 @@ def lst_bin_files(data_files, filetype='uvh5', input_cals=None, dlst=None, verbo
             continue
 
         # write to file
-        io.write_vis(bin_file, bin_data, bin_lst, freq_array, antpos, flags=flag_data, verbose=verbose, nsamples=num_data, filetype=filetype, **kwargs)
-        io.write_vis(std_file, std_data, bin_lst, freq_array, antpos, flags=flag_data, verbose=verbose, nsamples=num_data, filetype=filetype, **kwargs)
+        io.write_vis(bin_file, bin_data, bin_lst, freq_array, antpos, flags=flag_data, verbose=verbose, nsamples=num_data, filetype='uvh5', **kwargs)
+        io.write_vis(std_file, std_data, bin_lst, freq_array, antpos, flags=flag_data, verbose=verbose, nsamples=num_data, filetype='uvh5', **kwargs)
 
         del bin_file, std_file, bin_data, std_data, num_data, bin_lst, flag_data
```

Comment:
I don't think we need these dels now.

Reply:
I think these we could stand to keep. I'm not exactly sure when Python's garbage collector is invoked (automatically once a data structure hits zero references?) and b/c these are the biggest objects might as well make sure they are being removed before another batch of data is loaded in.